.gitignore

```diff
@@ -5,3 +5,5 @@ galene-password-generator/galene-password-generator
 passwd
 groups/*.json
 static/*.d.ts
+admin/admin
+admin/*.json
```

Comment:
Maybe we may also use here a `passwd` file format as Galène already choose this format for the admin interface?

Reply:
I made a code refactoring to make this change easier. But maybe we could make this an another issues because I don't know how to do that.

Reply:
The plan is that the `data/passwd` file will be replaced by a JSON file at some point, with different global users and user groups (so that you don't need to copy multiple identical user entries in multiple group files).